src/guides/v2.4/graphql/mutations/create-company-user.md

```diff
@@ -0,0 +1,179 @@
+---
+group: graphql
+title: createCompanyUser mutation
+contributor_name: Atwix
+contributor_link: https://www.atwix.com/
+b2b_only: true
+---
+
+Use the `createCompanyUser` mutation to create a new company user or add a user from the existing customer for your company.
+
+The `target_id` input attribute allows you to specify which node in the company structure will be the parent node of the company user. If you do not specify a value, the user will be assigned to the top-level (root) node of the company structure.
+
+You can get the `target_id` and the `role_id` with the [`company`]({{page.baseurl}}/graphql/queries/company.html) query.
+
+## Syntax
+
+```graphql
+mutation {
+    createCompanyUser(
+        input: CompanyUserCreateInput!
+    ) {
+        CreateCompanyUserOutput
+    }
+}
+```
+
+## Example usage
+
+The following example shows the minimal payload for adding a new customer to a customer's company.
```

Comment:
```suggestion
The following example shows the minimal payload to add a company user. Because a `targit_id` is not specified, Magento places the new company user at the top node of the company structure.
```